app/controllers/tags_controller.rb

```diff
@@ -2,7 +2,8 @@ class TagsController < ApplicationController
   before_filter :load_collection
   before_filter :check_user_status, :except => [ :show, :index, :show_hidden, :search, :feed ]
   before_filter :check_permission_to_wrangle, :except => [ :show, :index, :show_hidden, :search, :feed ]
-  before_filter :load_tag, :only => [:show, :edit, :update, :wrangle, :mass_update]
+  before_filter :load_tag, :only => [:edit, :update, :wrangle, :mass_update]
+  before_filter :load_tag_and_subtags, :only => [:show]
```

Comment:
Use the new Ruby 1.9 hash syntax.<br>Prefer `before_action` over `before_filter`.